src/components/registreringsInfo.tsx

```diff
@@ -0,0 +1,143 @@
+    }, [fnr]);
+
+    if (lasterRegistreringsdata) {
+        return (
+            <Panel border className="info_panel">
+                <Laster />
+            </Panel>
+        );
+    }
+
+    if (registreringHarFeil) {
+        return (
+            <Panel border className="info_panel">
+                <Errormelding />
+            </Panel>
+        );
+    }
+
+    const registrertAvNavn: StringOrNothing = registrering?.registrering?.manueltRegistrertAv?.enhet?.navn;
+    const registrertAvID: StringOrNothing = registrering?.registrering?.manueltRegistrertAv?.enhet?.id;
+    const registrertAvIdent: StringOrNothing = registrering?.registrering?.manueltRegistrertAv?.ident;
+    const datoRegistrert: StringOrNothing = registrering?.registrering?.opprettetDato;
+
+    const regDataHvorfor = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'dinSituasjon'
+    );
+    const hvorforSvar: StringOrNothing = regDataHvorfor?.svar;
+
+    const regDataSisteStilling = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'sisteStilling'
+    );
+    const sisteStillingSvar: StringOrNothing = regDataSisteStilling?.svar;
+
+    const regDataUtdanning = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'utdanning'
+    );
+    const UtdanningSvar: StringOrNothing = regDataUtdanning?.svar;
+
+    const regDataUtdanningGodkjent = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'utdanningGodkjent'
+    );
+    const UtdanningGodkjentSvar: StringOrNothing = regDataUtdanningGodkjent?.svar;
+
+    const regDataUtdanningBestatt = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'utdanningBestatt'
+    );
+    const UtdanningBestattSvar: StringOrNothing = regDataUtdanningBestatt?.svar;
+
+    const regDataHelse = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'helseHinder'
+    );
+    const HelseSvar: StringOrNothing = regDataHelse?.svar;
+
+    const regDataAnnet = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'andreForhold'
+    );
+    const AnnetSvar: StringOrNothing = regDataAnnet?.svar;
+
+    const regIdNavn = registrertAvID + ' ' + registrertAvNavn;
+    const regDato = 'Registrert: ' + formaterDato(datoRegistrert);
+    const regAv = 'Registrert av: ' + registrertAvIdent + ', ' + regIdNavn;
+
+    const regValues = [`${regDato}`, `${regAv}`];
+
+    const brukerRegistrering = registrering?.registrering;
+    const type = registrering?.type;
+
+    return (
+        <Panel border className="info_panel">
+            <Heading spacing level="2" size="large">
+                Registering
+            </Heading>
+            <DobbeltInformasjon header={regAv ? regAv : EMDASH} values={regValues ? regValues : [EMDASH]} />
```

Comment:
Er det meningen at det skal stå to ganger?
<img width="381" alt="bilde" src="https://github.com/navikt/veilarbdetaljerfs/assets/85442404/0d05db5d-c12a-4a45-8690-a15b9b4ea945">